src/nvc_info.c

```diff
@@ -337,8 +337,16 @@ find_path(struct error *err, const char *tag, const char *root, const char *targ
         char path[PATH_MAX];
         int ret;
 
-        if (path_resolve(err, path, root, target) < 0)
+        if (path_resolve(err, path, root, target) < 0) {
+                if (errno == EACCES) {
+                        // The current user doesn't have permission to access
+                        // this path. Treat this the same as a missing path.
+                        log_warnf("insufficient permissions for %s path %s", tag, target);
```

Comment:
I think an ideal log message would also contain EACCES (the specific error observed) and the system call that failed (the specific action that threw the error). But it's very fine.

Reply:
Although I don't disagree, getting access to the system call is not that straight forward. I also tend to tailor non-debug log messages more at the user where mentioning `EACCES` may not have as much value.

If you don't feel too strongly about this, I will leave it as is for now.